doc/source/index.rst

```diff
@@ -97,6 +97,11 @@ Limitations
 Note: MAPDL models that have not been preprocessed with ACP can be converted. For more
 information, see `Import of Legacy Mechanical APDL Composite Models`_ in the Ansys Help.
 
+API Changes
+'''''''''''
+These are the incompatible changes in the API of PyDPF - Composites and the underlying DPF operators:
+  - With DPF server 10.0 (2025 R2), the input pin ``fields_container`` of the ``composite::multiple_failure_criteria_operator`` is named ``section_data_container``.
```

Comment:
```suggestion
  - With DPF server 10.0 (2025 R2), the input pin ``fields_container`` of the ``composite::multiple_failure_criteria_operator`` is renamed to ``section_data_container``.
```
To be more specific which one is the new name.

Do we also need to add some try / except in our examples so that this works with multiple backend versions?